### ISSUE TITLE
Interpolation as property or selector has format bug

### ISSUE THREAD

**Issue by ctuu** [TheRealSyler/sass-formatter#5]
**Describe the bug**
I try to format this code.
```
#{body}
    color: red

#{main}
    color:red
```

it delete the Curly Brackets and something strange.
```
#
    body
    color: red

    &main
        color:red
```

**Additional context**
it also show "Curly Brackets Are Not AllowedSass" message at problem panel.

BTW: the README.md at your [sass - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=Syler.sass-indented) can't show all the badge because some extra text at the badge URL.

**Comment by TheRealSyler** [comment]
@ctuu i already know why it happens, i will fix it in the next version.
about the badges, i tried to fix it but there is only so much energy that i want to spend on a problem like that.

**Comment by TheRealSyler** [comment]
oh and to disable the message disable the linter in the sass options, it should be disabled by default, the linter is a project that i kind of abandoned.

**Comment by TheRealSyler** [comment]
fixed in 52090bb0f0e848ecd0d88af93d09a0c3ffe4194a, it might take a while until its added to the extension.
